src/styles.css

```diff
@@ -7,10 +7,22 @@ pre {
 
 .branding img {
   height: 2rem;
+  vertical-align: middle;
+  width: 36px;
```

Comment:
why not rem?

Reply:
Got it from the clarity.design - but I guess rem could work better